lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,316 @@
+from django.core.management.base import BaseCommand, CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from django.test import TestCase
+
+from .. import rename_user
+
+
+class TestRenameUser(TestCase):
+    NEW_USERNAME = "targetName"
+
+
+    @classmethod
+    def setUpClass(cls):
+        client = MongoClient()
+        #chooses to temporarily write to the forum database
+        cls.db = client.forum
+        #maintain arrays of mongo data so that we can delete them when the tests are done
+        cls.mongo_user_ids = []
+        cls.mongo_comment_ids = []
+        cls.sql_users = []
+        cls.rename_cmd = rename_user.Command()
+        cls.seed_sql_users()
+        cls.seed_mongo_comments()
+        cls.seed_mongo_users()
+
+
+    @classmethod
+    def seed_sql_users(cls):
+        cls.sql_users.append(UserFactory.create())
+        cls.sql_users.append(UserFactory.create())
+        cls.sql_users.append(UserFactory.create())
+
+    @classmethod
+    def seed_mongo_comments(cls):
+        comment_1 = {
```

Comment:
again, just a formatting comment.  `paver run_qualty` or `paver run_pep8` should help you flag.  in general you probably want something like
```
def seed_mongo_comments(cls):
    comment_1 = {
        "_type": "CommentThread",
        "abuse_flaggers": [],
...
```
4 spaces per indent